specificatie/openapi.yaml

```diff
@@ -1032,11 +1044,14 @@ components:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/DatumOnvolledig"
     OpschortingBijhoudingBasis:
       type: "object"
+      description: |
+                    * **reden** - wordt gevuld op bassi van de waarden die voorkomen in de tabel 'redenopschortingbijhouding' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
Gezien de opmerking van @fsamwel ('De huidige landelijke tabellen hebben een code met onderscores') vraag ik me af of de tabelnaam 'redenopschortingbijhouding' wel de juiste naam is.

"bassi" --> "basis"